### PR TITLE
Cleaning up Flight Director etcd Values

### PR DESCRIPTION
**Pull request by matthewdfuller**
Changing "FD/" to "flight-director/", lowercasing values.

### CODE REVIEW

File: v3/fleet-local/control/flight-director@.service [hunk 1]
```diff
@@ -19,31 +19,31 @@ ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
   --name flight-director \
   --net='host' \
   -e LOG_APP_NAME=flight-director \
-  -e FD_API_SERVER_PORT=`etcdctl get /FD/FD_API_SERVER_PORT` \
-  -e FD_CHRONOS_MASTER=`etcdctl get /FD/FD_CHRONOS_MASTER` \
-  -e FD_DB_DATABASE=`etcdctl get /FD/FD_DB_DATABASE` \
-  -e FD_DB_ENGINE=`etcdctl get /FD/FD_DB_ENGINE` \
+  -e FD_API_SERVER_PORT=`etcdctl get /flight-director/api-server-port` \
```
Comment (chr0n1x) [inline v3/fleet-local/control/flight-director@.service]: I would move all of these into `/flight-director/config`. The `/flight-director` namespace is actually created & used by the app.

**Comment by anshupande** [comment]
looks much cleaner and good to see convention in place :+1:

File: v3/setup/images-control.sh [hunk 2]
```diff
@@ -12,18 +12,18 @@ if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
...
 etcdctl set images-control-bootstrapped true
 
-etcdctl set /images/chronos      "mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/fd           "behance/flight-director:latest"
-etcdctl set /images/marathon     "mesosphere/marathon:v0.15.1"
-etcdctl set /images/mesos-master "mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/zk-exhibitor "behance/docker-zk-exhibitor:latest"
-etcdctl set /images/cfn-signal   "behance/docker-cfn-bootstrap:latest"
+etcdctl set /images/chronos      	"mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
+etcdctl set /images/flight-director "behance/flight-director:latest"
```
Comment (chr0n1x) [inline v3/setup/images-control.sh:16]: whitesppaaaaace aliggggn :P

**Comment by matthewdfuller** [comment]
Note to self: don't merge until `infrastructure` is updated too

**Comment by matthewdfuller** [comment]
Naming Possibilities (vote for your favorite)
- `/flight-director/config/` (vote with a thumbs up)
- `/flight-director-fleet/config/` (vote with a heart)

**Comment by chr0n1x** [comment]
lol cc @iderdik

**Comment by matthewdfuller** [comment]
A/B'd and it looks good to me

```
core@ip-10-74-131-180 sandbox-rabbit-us-east-1d-control ~ $ etcdctl ls /flight-director/config
/flight-director/config/marathon-master-protocol
/flight-director/config/db-name
/flight-director/config/db-username
/flight-director/config/dockercfg-location
/flight-director/config/debug
/flight-director/config/fixtures
/flight-director/config/log-location
/flight-director/config/db-path
/flight-director/config/api-server-port
/flight-director/config/event-interface
/flight-director/config/event-port
/flight-director/config/log-marathon-api-calls
/flight-director/config/mesos-master
/flight-director/config/mesos-master-protocol
/flight-director/config/authorizer-type
/flight-director/config/chronos-master
/flight-director/config/db-engine
/flight-director/config/kv-server
/flight-director/config/log-level
/flight-director/config/marathon-master
/flight-director/config/allow-marathon-unverified-tls
```

File: v3/setup/images-control.sh [hunk 3]
```diff
@@ -12,18 +12,18 @@ if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
...
+etcdctl set /images/marathon     	"mesosphere/marathon:v0.15.1"
+etcdctl set /images/mesos-master 	"mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
```
Comment (iderdik) [inline v3/setup/images-control.sh:18]: Can we make these the latest tested `mesosphere/mesos-master:0.28.1` and `mesosphere/marathon:v1.1.1`

Comment (matthewdfuller) [inline v3/setup/images-control.sh:18]: Yes, but not in this PR 😄

**Comment by iderdik** [comment]
👍

**Comment by matthewdfuller** [comment]
Must be merged with: https://github.com/adobe-platform/infrastructure/pull/61

**Comment by chr0n1x** [comment]
MERGE MERGE MERGEEEEEE

**Pull request merged**